src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,359 @@
+Replace `extendPlugins: 'defaults'` with `extendMarkdownConfig: false` and add the [separate options for GitHub-Flavored Markdown and SmartyPants](/en/guides/integrations-guide/mdx/#options-inherited-from-markdown-config) to enable these default plugins individually in MDX.
+
+```js del={6} ins={7-9}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  mdx: {
+    extendPlugins: defaults,
+    extendMarkdownConfig: false,
+    smartypants: true,
+    gfm: true,
+  }
+});
+```
+
+#### Added: More MDX config options to match Markdown
+
+Setting `extendMarkdownConfig: false` now allows you to individually set [every available Markdown configuration option](/en/reference/configuration-reference/#markdown-options) (except `drafts`) separately in your MDX integration configuration.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: 'prism',
+    remarkPlugins: [remarkPlugin1],
+    gfm: true,
+  },
+  integrations: [
+    mdx({
+      // `syntaxHighlight` inherited from Markdown
+
+      // Markdown `remarkPlugins` ignored,
+      // only `remarkPlugin2` applied.
+      remarkPlugins: [remarkPlugin2],
+      // `gfm` overridden to `false`
+      gfm: false,
+    })
+  ]
+});
+```
+
+##### What should I do?
+
+Revisit your Markdown and MDX configuration and compare your existing config with the new options available.
+
+:::tip[remember]
+- Astro's two default Markdown plugins are now individually configurable, and are not removed when you add other plugins. No more extending defaults!
+- The option to extend your Markdown plugin config to MDX has moved from a selection (just defaults or all plugins) to toggle (all or none).
+- Every Markdown config option (except drafts) can also be configured independently in MDX!
+:::
+
+### Added: Plugin access to frontmatter
+
+In 1.0, remark and rehype plugins did not have access to user frontmatter. Astro merged plugin frontmatter with your file's frontmatter, without passing the file frontmatter to your plugins.
+
+Astro 2.0 gives remark and rehype plugins access to user frontmatter via frontmatter injection. This allows plugin authors to modify a user's existing frontmatter, or compute new properties based on other properties.
+
+
+#### What should I do?
+
+Check any remark and rehype plugins you have written to see whether their behavior has changed. Note that `data.astro.frontmatter` is now the _complete_ Markdown or MDX document's frontmatter, rather than an empty object.
+
+### Reserved: `src/content/`
+
+In 1.0, the [Collections API was hidden behind an experimental flag](#astro-20-experimental-flags-removed). This API reserves `src/content/` as a special folder.
+
+Astro 2.0 now includes the Collections API in the core codebase for organizing your Markdown and MDX files into [content collections](/en/guides/content-collections/).
```

Comment:
Do you think the history is important here given how relatively brief the experimental period was? Not sure it’s particularly relevant to users.

```suggestion
Astro 2.0 now includes the Collections API for organizing your Markdown and MDX files into [content collections](/en/guides/content-collections/). This API reserves `src/content/` as a special folder.
```

Reply:
Right. I was trying to think of a "previously" statement to match the pattern of the others. But, no need to force it!